src/components/TagsPoplist/index.tsx

```diff
@@ -33,19 +32,37 @@ const StyledTooltipReference = styled(TooltipReference)`
   padding-right: 8px;
 `
 
+const StyledTagContainer = styled.div`
+  display: flex;
+  align-items: center;
+  color: ${({ theme }) => theme.colors.gray700};
+`
+
+const StyledManyTagsContainer = styled.div`
+  box-shadow: 0 -1px 5px 3px rgba(165, 165, 205, 0.15);
+  padding: ${({ theme }) => `${theme.space['0.5']} ${theme.space['1']}`};
+  display: flex;
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.white};
+  border-radius: ${({ theme }) => theme.radii.default};
+  max-width: 80vw;
+  flex-wrap: wrap;
+`
+
 export type TagsPoplistProps = {
   maxLength?: number
   maxTagWidth?: number
   tags?: string[]
   threshold?: number
+  css: string | undefined
 }
 
 const TagsPoplist = ({
   maxLength = 600,
   maxTagWidth = 115,
   tags = [],
   threshold = 1,
-  ...props
+  css,
```

Comment:
This won't be interpreted, css is a runtime variable, same as key for exemple :)